### PR TITLE
refactor(markdown in json): replace PagesJsonBlockMarkdown with gatsby-plugin-json-remark

### PR DESCRIPTION
**Pull request by mirabilio**
- install `gatsby-plugin-json-remark` plugin with configuration:
  - ```paths: [`${__dirname}/content/pages`], ```
     ```pathsExclude: [`${__dirname}/content/pages/contact.json`],```
     ```fieldNameBlacklist: ['id', 'children', 'parent', 'fields', 'internal', 'path', 'template'], ```
- remove onCreateNode function from `gatsby-node.js`
- remove `childrenPagesJsonBlockMarkdown` block from graphql query in `page.js`
- add fields `titleHtml`, `hero.headlineHtml`, `hero.textLineHtml`, `hero.ctas.labelHtml`, `blocks.contentHtml` to graphql query in `page.js`
- add *Html placeholder fields to `dummy.json`
- add `.vscode/ ` and `*.code-workspace` to `.gitignore`

### CODE REVIEW

**Comment by spbyrne** [comment]
Thanks @mirabilio! Much appreciated. I'll take a look & test this ASAP.

**Comment by spbyrne** [comment]
Hey @mirabilio, finally got a chance to test; looks good! This is a significant improvement. I knew there had to be a better way.

File: yarn.lock [hunk 1]
```diff
@@ -6651,7 +6662,11 @@ gatsby-plugin-styled-components@^3.1.11:
   integrity sha512-utEcQJno6/eP6+LB21cUsgwW2fXMItvlOuw0vIRDXA8PnewEzhHSN2i4BuramQbBz3KTVUnxygtTC5Wo1lDcUA==
   dependencies:
     "@babel/runtime" "^7.8.7"
+<<<<<<< HEAD
+
+=======
     
+>>>>>>> upstream/master
```
Comment (parthsdoshi) [inline yarn.lock:6669]: this doesn't seem right

File: yarn.lock [hunk 2]
```diff
@@ -14666,7 +14696,11 @@ tinacms@^0.19.1:
     moment ">=2 <=3"
     react-datetime "^2.16.3"
     react-tinacms-editor "^0.5.1"
+<<<<<<< HEAD
+
+=======
     
+>>>>>>> upstream/master
```
Comment (parthsdoshi) [inline yarn.lock:14703]: this too

**Pull request merged**